File: Model/conseguenzacollisione.cpp
#include "conseguenzacollisione.h"

ConseguenzaCollisione::ConseguenzaCollisione(unsigned int e, double m): energia(e), magnitudo(m){}

unsigned int ConseguenzaCollisione::getEn() const{
    return energia;
}

double ConseguenzaCollisione::getMa() const {
    return magnitudo;
}

File: Java/src/astrokalk/ConseguenzaCollisione.java
package astrokalk;

public class ConseguenzaCollisione {

    private long energia;
    private double magnitudo;


    public ConseguenzaCollisione(long e, double m){
        energia = e;
        magnitudo = m;
    }

    public long getEn(){
        return energia;
    }

    public double getMa(){
        return magnitudo;
    }

}
